PROJECT_CHARTER_TEMPLATE.md

```diff
@@ -0,0 +1,77 @@
+# ${PROJECT} Charter
+
+_note: the purpose of a project charter is to provide a brief introduction_
+_to the project from a technical, community, or business perspective. The_ 
+_document also connects a project's community leadership and governance with the_
+_OpenJS Foundation's governance._
+
+
+## Section 0: Guiding Principles (optional)
+
+_directions: provide a concise, high-level statement about_
+_the project's long-term principles, values, or mission._
+
+## Section 1: Scope
+
+_directions: Include a 3-4 sentence summary of what the project does,_
+_and/or what problems it solves. Imagine trying to explain your work_
+_to a colleague who is familiar with related technical concepts but unfamiliar_
+_with the project. You may also want to describe the project's value to community_
+_and/or business stakeholders._
+
+### 1.1: In-scope (optional)
+
+_directions: list or bullet out problem spaces, use cases, repositories_
+_or other projects which are included with the work but may not be readily_
+_apparent. This may help differentiate the project from other solutions in the_
+_space._
+
+### 1.2: Out-of-Scope (optional)
+
+_directions: list or bullet out areas that may be seen to be related but are_
+_not included in the scope of this project. This may help clarify the kind of_
+_features, contributions, issues or problems the project is looking for._
+
+## Section 2: Relationship with OpenJS Foundation CPC.
+
+_directions: describe how the project intersects with the Cross Project_
+_Council._ 
```

Comment:
I’m not sure if it’s clear what to fill in this section.

Reply:
The Node TSC charter has these 2 paragraphs under its Section 2 about the relationship between the Node TSC and the CPC, I think they're good paragraphs & was imagining something like that would go here. I agree it's not super clear as written!

Reply:
Perhaps the template would benefit from having links to good examples, such as this one that you mention from the Node.js TSC charter https://github.com/nodejs/TSC/blob/master/TSC-Charter.md#section-2-evolution-of-openjs-foundation-governance